streams/shared/src/main/scala/zio/stream/experimental/ZStream.scala

```diff
@@ -3774,9 +3814,10 @@ class ZStream[-R, +E, +A](val channel: ZChannel[R, Any, Any, Any, E, Chunk[A], A
   /**
    * Threads the stream through the transformation function `f`.
    */
-  final def via[R2, E2, A2](f: ZStream[R, E, A] => ZStream[R2, E2, A2])(implicit
+  final def via[R2 <: R, E2 >: E, A2 >: A, A3](pipeline: ZPipeline[R2, E2, A2, A3])(implicit
```

Comment:
I think we may want to keep the other variant too to help migration, as it also existed on the old `ZStream`

Reply:
Yeah that makes sense.

Reply:
Done